doc/source/tutorials/VectorFitting.ipynb

```diff
@@ -167,17 +167,17 @@
    "metadata": {},
    "source": [
     "## References\n",
-    "<a id='link_ref1'>[1]</a> B. Gustavsen and A. Semlyen, \"Rational approximation of frequency domain responses by vector fitting,\" in IEEE Transactions on Power Delivery, vol. 14, no. 3, pp. 1052-1061, July 1999, DOI: [10.1109/61.772353](https://doi.org/10.1109/61.772353).\n",
+    "<div id='link_ref1'></div>[1] B. Gustavsen and A. Semlyen, \"Rational approximation of frequency domain responses by vector fitting,\" in IEEE Transactions on Power Delivery, vol. 14, no. 3, pp. 1052-1061, July 1999, DOI: [10.1109/61.772353](https://doi.org/10.1109/61.772353).\n",
```

Comment:
Thanks for fixing.
Still, I find it strange that this syntax was wrong, because I created the link with the Jupyter GUI. Or is this more like a Sphinx thing?

Reply:
it works indeed under Jupyter, but no more after the conversion into html... so maybe, yes.